module/observations/helper.js

```diff
@@ -2322,6 +2322,39 @@ module.exports = class ObservationsHelper {
   
       lastSubmissionNumber = lastSubmissionForObservationEntity.result + 1;
   
+      let programInformation = null;
+
+      if(solutionDocument.programId){
+
+      let programQueryObject = {
+        _id: solutionDocument.programId,
+        status: "active"
```

Comment:
@borkarsaish65 do not directly use active use common constant key. it will be already present

Reply:
resolved